src/script/page/RightSidebar/ConversationDetails/ConversationDetails.tsx

```diff
@@ -0,0 +1,426 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {RECEIPT_MODE} from '@wireapp/api-client/src/conversation/data/';
+import cx from 'classnames';
+import {FC, useEffect, useRef, useState} from 'react';
+
+import Icon from 'Components/Icon';
+import ServiceDetails from 'Components/panel/ServiceDetails';
+import ServiceList from 'Components/ServiceList';
+import PanelActions from 'Components/panel/PanelActions';
+import UserSearchableList from 'Components/UserSearchableList';
+
+import {registerReactComponent, useKoSubscribableChildren} from 'Util/ComponentUtil';
+import {t} from 'Util/LocalizerUtil';
+import {sortUsersByPriority} from 'Util/StringUtil';
+import {formatDuration} from 'Util/TimeUtil';
+
+import getConversationActions from './components/getConversationActions';
```

Comment:
Let's maybe move it to a `util` folder, rather than in `components`. 

`page/RightSidebar/utils/getConversationActions`